README.rst

```diff
@@ -8,6 +8,15 @@
 .. image:: https://codecov.io/gh/nipy/dipy/branch/master/graph/badge.svg
   :target: https://codecov.io/gh/nipy/dipy
 
+.. image:: https://img.shields.io/github/release/qubyte/rubidium.svg
+  :target: https://github.com/nipy/dipy/releases/latest
+
+.. image:: https://img.shields.io/pypi/pyversions/Django.svg
+  :target: https://pypi.python.org/pypi/dipy
```

Comment:
is it really necessary to show python version? IMHO, you can remove it

Reply:
Again, my fault:it was showing Django's. I think knowing which python versions are supported is worthwhile, but looks like that is only possible for PyPI, and it is not working wither for DIPY, so I'm removing it.